linbox/ring/ntl/ntl-lzz_p.h

```diff
@@ -340,10 +340,10 @@ namespace LinBox
 
 		UnparametricRandIter<NTL::zz_p> (const NTL_zz_p & F,
                                          const integer& size=0,
-                                         const integer& seed=0) :
+                                         const uint64_t& seed=0) :
                 _size(size), _seed(seed), _ring(F)
             {
-                if (_seed == integer(0)) _seed = int64_t(time(NULL));
+                if (_seed == 0) _seed = int64_t(time(NULL));
```

Comment:
why not uint64_t(time(NULL)) ?
Note, that there is also Givaro::BaseTimer::seed() ...

Reply:
I did remove that cast